lib/matplotlib/backends/_backend_tk.py

```diff
@@ -507,16 +507,21 @@ class NavigationToolbar2Tk(NavigationToolbar2, tk.Frame):
     Attributes
     ----------
     canvas : `FigureCanvas`
-        the figure canvas on which to operate
+        The figure canvas on which to operate.
     win : tk.Window
-        the tk.Window which owns this toolbar
-
+        The tk.Window which owns this toolbar.
+    pack_toolbar : bool, default: True
+        If True, add the toolbar to the parent's pack manager's packing list
+        during initialization with *side='bottom'* and *fill='x'*.
+        If you want to use the toolbar with a different layout manager, use
+        *pack_toolbar=False*.
     """
-    def __init__(self, canvas, window):
+    def __init__(self, canvas, window, pack_toolbar=True):
```

Comment:
please make the argument keyword only

Reply:
After some googling the signature is now:

`def __init__(self, canvas, window, *, pack_toolbar=True):`

I didn't know this was possible at all, so please correct me if there is a more canonical way to do it.